rasa/core/policies/ted_policy.py

```diff
@@ -383,17 +602,16 @@ def persist(self, path: Text) -> None:
         io_utils.pickle_dump(
             model_path / f"{SAVE_MODEL_FILE_NAME}.meta.pkl", self.config
         )
-        io_utils.json_pickle(
+        io_utils.pickle_dump(
```

Comment:
why we cannot use `io_utils.json_pickle`?

Reply:
Because we can have `coo_matrix` in our `data_example` and it is not json-serializale